helmfile/charts/gatekeeper-templates/templates/restrict-latest-tag/template.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
```

Comment:
Shouldn't this also be wrapped in `{{- if .Values.restrictTags.enable -}}`?

Reply:
Thanks Fredrik !! I have fixed now

Reply:
> Shouldn't this
This we don't need as we have not wrapped for previous 3 policies too !!